.php_cs

```diff
@@ -86,38 +87,37 @@ $config->setRules(
         'no_unneeded_final_method' => true,
         'no_unused_imports' => true,
         'no_useless_else' => true,
-        'no_useless_return' => true,
         'no_whitespace_before_comma_in_array' => true,
         'no_whitespace_in_blank_line' => true,
-        'non_printable_character' => ['use_escape_sequences_in_strings' => true],
         'normalize_index_brace' => true,
+        'object_operator_without_whitespace' => true,
         'ordered_imports' => true,
         'phpdoc_add_missing_param_annotation' => true,
         'phpdoc_indent' => true,
         'phpdoc_no_access' => true,
-        'phpdoc_no_empty_return' => true,
         'phpdoc_order' => true,
         'phpdoc_return_self_reference' => true,
         'phpdoc_scalar' => true,
         'phpdoc_separation' => true,
         'phpdoc_single_line_var_spacing' => true,
         'phpdoc_trim' => true,
         'phpdoc_types' => true,
-        'pow_to_exponentiation' => true,
```

Comment:
👎

Reply:
I'm generally OK with this transformation except it's a risky rule. `pow()`/`**` isn't used that often anyway, so I'd leave it for users to opt-in.

Reply:
Again, risky when someone is doing insane things (overriding `pow`)